json/src/value.rs

```diff
@@ -327,6 +395,86 @@ impl Value {
     }
 }
 
+/// A type that can be used to index into a `serde_json::Value`. See the `get`
+/// and `get_mut` methods of `Value`.
+pub trait Index: private::Sealed {
```

Comment:
Here is what the rustdoc looks ilke:


![selection_018](https://cloud.githubusercontent.com/assets/1940490/22250258/1948a970-e1fb-11e6-91cf-f7a189ee64ec.png)